src/hooks/core/chat/useSystemMessage.ts

```diff
@@ -120,7 +120,12 @@ const useSystemMessage = () => {
         continue;
       }
 
-      if (checkFirstMessage?.text?.toLocaleLowerCase()?.includes('you joinedt this community')) {
+      if (checkFirstMessage?.text?.toLocaleLowerCase()?.includes('you joined this community')) {
+        if (isMe(checkFirstMessage?.system_user)) {
+          firstMessage = checkFirstMessage;
+          break;
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `isMe(checkFirstMessage?.system_user)` is used to check if the system user is the current user. However, it's not clear what the function `isMe` does as it's not defined in this hunk. If it checks whether the user is the system user, then it might be redundant because `checkFirstMessage?.system_user` should already be a boolean indicating whether the message is from the system user or not. If `isMe` has other functionalities, please disregard this comment.

```diff
-        if (isMe(checkFirstMessage?.system_user)) {
+        if (checkFirstMessage?.system_user) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->